randovania/games/samus_returns/db_integrity.py

```diff
@@ -0,0 +1,43 @@
+from collections.abc import Iterator
+
+from randovania.game_description.game_description import GameDescription
+from randovania.game_description.integrity_check import (
+    check_for_items_to_be_replaced_by_templates,
+    check_for_resources_to_use_together,
+)
+
+# Short name resource -> recommended template name
+use_templates_over_items = {
+    "Spring": "Use Spring Ball",
+    "Spider": "Use Spider Ball",
+    "Bomb": "Lay Bomb or Lay Any Bomb",
+    "PBAmmo": "Lay Power Bomb or Lay Any Bomb",
+    "Wave": "Shoot Wave Beam",
+    "Spazer": "Shoot Spazer Beam",
```

Comment:
Plasma too or nah? Feels a but weird to have templates for anybeam except last, but i could live with it.

Reply:
The templates are there because of how beam progression works. Plasma is last so it has no special case. But, with split beams coming soon there's no real need for the templates anymore.